backend/spec/features/admin/products/edit/images_spec.rb

```diff
@@ -64,6 +64,8 @@
           expect(page).to have_xpath("//img[contains(@src,'ror_ringer')]")
         end
       end
+
+      expect(Spree::Image.last.viewable).to eq(product.master)
```

Comment:
Err. Is there a way we can make this assertion without looking into the db here?